airflow_config/scripts/initairflow.sh

```diff
@@ -18,6 +18,8 @@
 : "${AIRFLOW_ROLE:="Admin"}"
```

Comment:
I wonder if maybe we should migrate these deployment scripts to python. It's possible we would be able to reduce the size and make them more flexible. What do you think?